src/gnmi_clients/gnoi_client.go

```diff
@@ -16,38 +16,41 @@ import (
 	"encoding/json"
 	"strings"
 	"google.golang.org/grpc/metadata"
+	"github.com/google/gnxi/utils/credentials"
 )
 
 var (
 	module = flag.String("module", "System", "gNOI Module")
 	rpc = flag.String("rpc", "Time", "rpc call in specified module to call")
 	target = flag.String("target", "localhost:8080", "Address:port of gNOI Server")
 	args = flag.String("jsonin", "", "RPC Arguments in json format")
-	username = flag.String("username", "", "Username if required")
-	password = flag.String("password", "", "Password if required")
+	// username = flag.String("username", "", "Username if required")
```

Comment:
Why comment out lines rather than deleting them?

Reply:
That was an oversight. I was adding the user/pass manually, but now using the gnxi/utils/credentials library that provides al lthe same flags as gnmI_get etc, which includes user/pass. I will remove it.

Reply:
removed.